cmake/PackageExtensionTemplate.cmake

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.3)
+project(LanternDB)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+find_package(PostgreSQL REQUIRED)
+
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/
```

Comment:
Is this a standalone cmake project file that will be invoked separately?
Why are these not in the maim CMakeLists.txt?

I think it is typical that `cmake/` folder only contains helper modules/function for the main CMakeLists file. But maybe I am missing something here.